### PR TITLE
feature/mx-1406 prepare preview implementation

### PR DESCRIPTION
**Pull request by cutoffthetop**
# PR Context
- prep PR to keep https://github.com/robert-koch-institut/mex-backend/pull/114 lighter

### Changes
- BREAKING: move `to_primitive` to a more fittingly named `mex.backend.serialization`
- BREAKING: swap `INDEXABLE_MODEL_CLASSES_BY_NAME` for `ALL_MODEL_CLASSES_BY_NAME`
  to also include non-indexable models (namely: merged models)
- BREAKING: rename `fetch_extracted_data` to a more consistent `fetch_extracted_items`
- harmonize PagedAuxiliaryResponse with Merged- and ExtractedItemSearchResponse
- use PlainSerializer instead of JSONResponse to fix serializing with a happy mypy
- move searching and fetching of extracted and merged items to `helpers` module
  so they can be reused more easily outside of the endpoint code
- use starlette status code constants instead of plain integers for readability

### Removed

- remove redundant `status_code=200` config on endpoints
- drop unused `UnprefixedType` support for `entityType` parameters

### Fixed

- fix empty graph search results not returning empty lists but raising errors
- move `purge-db` from entrypoints to pdm scripts, because it's not part of the module

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,12 +11,28 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changes
 
+- BREAKING: move `to_primitive` to a more fittingly named `mex.backend.serialization`
+- BREAKING: swap `INDEXABLE_MODEL_CLASSES_BY_NAME` for `ALL_MODEL_CLASSES_BY_NAME`
+  to also include non-indexable models (namely: merged models)
+- BREAKING: rename `fetch_extracted_data` to a more consistent `fetch_extracted_data`
```
Comment (mr-kamran-ali) [inline CHANGELOG.md:17]: these are the same names. I think you wanted `fetch_extracted_items` here.

File: mex/backend/extracted/helpers.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+from mex.backend.extracted.models import ExtractedItemSearch
+from mex.backend.graph.connector import GraphConnector
+
+
+def search_extracted_items_in_graph(
```
Comment (mr-kamran-ali) [inline mex/backend/extracted/helpers.py:5]: shouldnt search result in multiple items especially when `skip` and `limit` is expected.

**Review by mr-kamran-ali — COMMENTED** [review]
almost nothing found. just one typo and one clarification.

File: CHANGELOG.md (hunk 1 above)
Comment (cutoffthetop) [reply CHANGELOG.md]: ups, yeah ad63e9b

File: mex/backend/extracted/helpers.py (hunk 2 above)
Comment (cutoffthetop) [reply mex/backend/extracted/helpers.py:5]: it does. `ExtractedItemSearch` is just a wrapper model, and it has `items` which is a list of extracted items

Comment (mr-kamran-ali) [reply mex/backend/extracted/helpers.py:5]: ah yeah, I see it.

**Review by mr-kamran-ali — APPROVED** [review]
🚀

**Pull request merged**